native/src/components/MapView.tsx

```diff
@@ -12,49 +14,79 @@ const MapContainer = styled.View`
 const StyledMap = styled(MapboxGL.MapView)`
   width: 700px;
   height: 500px;
+  z-index: 1;
 `
 
 type MapViewPropsType = {
   boundingBox: BBox
   featureCollection: FeatureCollection
+  currentFeature?: Feature<Point>
 }
 
 const textOffsetY = 1.25
-const layerProps: SymbolLayerProps = {
-  id: 'point',
-  style: {
-    symbolPlacement: 'point',
-    iconAllowOverlap: true,
-    iconIgnorePlacement: true,
-    iconImage: ['get', 'symbol'],
-    textField: ['get', 'title'],
-    textFont: ['Roboto Regular'],
-    textOffset: [0, textOffsetY],
-    textAnchor: 'top',
-    textSize: 12
-  }
-}
+const layerId = 'point'
 
 // Has to be set even if we use map libre
 MapboxGL.setAccessToken(mapConfig.accessToken)
-const MapView = ({ boundingBox, featureCollection }: MapViewPropsType): ReactElement => {
+const MapView = ({ boundingBox, featureCollection, currentFeature }: MapViewPropsType): ReactElement => {
+  const ref = React.useRef<MapboxGL.MapView | null>(null)
+  const [activeFeature, setActiveFeature] = useState<Feature<Point> | null>(currentFeature ?? null)
+  const [showPopup, setShowPopup] = useState<boolean>(false)
+  const layerProps: SymbolLayerProps = {
+    id: layerId,
+    style: {
+      symbolPlacement: 'point',
+      iconAllowOverlap: true,
+      iconIgnorePlacement: true,
+      iconImage: ['get', 'symbol'],
+      textField: ['case', ['==', ['get', 'id'], activeFeature?.properties?.id ?? -1], ['get', 'title'], ''],
```

Comment:
I didn't find a better approach than the buildIn expression to render the text field only for activePoi.
This may cause the fading in emulator. On native device everything works fine!

Reply:
I think I would either not show a title at all or do it the other way round, i.e. only not show the title for the selected feature as the title is shown in the popup anyway and I don't see a reason why we need it twice.

Reply:
ok then we keep it for now shown for all and we can discuss about this